### PR TITLE
📝 Initial contributors and developer setup guides

### LINKED ISSUE #3: 📝 Add contributing guide to set dev standards

### LINKED ISSUE #4: 👨🏻‍💻 Add developer guide for env setup

### PR DESCRIPTION
**Pull request by znatty22**
Closes #3 
Closes #4

### CODE REVIEW

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+After you've made the change you want on your local machine its time 
+to commit that change to the project repo. 
+
+All commits are pre-prepended with a standard emoji. 
+
+### Commit Standards 
+
+| Short Code          | Definition    | Renders As |
+----------------------| ------------  |------------|
+|:sparkles:           | New Feature   |     ✨     |
```
Comment (haynescd) [inline docs/contributing.md:40]: I think you need to escape these by wrapping them in ''

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+|:refactor:           | Refactor      |     ♻️      |
```
Comment (haynescd) [inline docs/contributing.md:41]: Should be ':recycle:'

File: docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+|:bug:                | Bug           |     🐛     |
+|:memo:               | Documentation |     📝     |
+|:white_check_mark:   | Testing       |     ✅     |
+|:construction_worker:| DevOps        |     👷     |
+
+
+When commits are pushed to Github, Github renders them as emojis. This makes it 
+easy to visually browser through commit history and get a quick idea of the
```
Comment (haynescd) [inline docs/contributing.md:49]:
```suggestion
easy to visually browse through commit history and get a quick idea of the 
```

File: docs/contributing.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+Once you're happy with your changes, its time to [open a Pull Request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request?tool=webui)
+so that others can review your code and upon their approval you can merge your branch
+into the main branch.
+
+### Title
+Similar to commit messages, you should prefix the PR title with the appropriate emoji
+that relates to the type of change it is (feature, refactor, etc). 
+
+For example:
+**:sparkles: Add new jira cmd to cli**
```
Comment (haynescd) [inline docs/contributing.md:102]:
```suggestion
**':sparkles:' Add new jira cmd to cli**
```

File: docs/deveoper-guide.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+[pyenv](https://realpython.com/intro-to-pyenv/) to manage python versions and
+virtual environments.
+
+Follow the [installation instructions here](https://realpython.com/intro-to-pyenv/#installing-pyenv)
+to install pyenv.
+
+Next install python using pyenv:
+
+```shell
+# install latest version of python 3.11
```
Comment (haynescd) [inline docs/deveoper-guide.md:26]: Did you want to upgrade to 3.13? just released hahah

**Review by haynescd — CHANGES_REQUESTED** [review]

File: docs/deveoper-guide.md (hunk 5 above)
Comment (znatty22) [reply docs/deveoper-guide.md:26]: I know 3.12 is the latest stable release, but I'm worried about other libraries not supporting the latest stable release. So I went with 3.11 but maybe that is silly and we should go for 3.12 at least 🤷‍♀️  What do you think? @haynescd

File: docs/contributing.md (hunk 1 above)
Comment (znatty22) [reply docs/contributing.md]: I had to use `` bc '' didn't work 🤷‍♀️

File: docs/deveoper-guide.md (hunk 5 above)
Comment (haynescd) [reply docs/deveoper-guide.md:26]: Looks like 3.12 has a big improvement to asyncio `The [asyncio](https://docs.python.org/3/library/asyncio.html#module-asyncio) package has had a number of performance improvements, with some benchmarks showing a 75% speed up.`

So maybe worth the upgrade.

**Review by haynescd — APPROVED** [review]

File: docs/deveoper-guide.md (hunk 5 above)
Comment (znatty22) [reply docs/deveoper-guide.md:26]: Alright lets doooo it

**Pull request merged**